src/java.base/share/classes/java/security/KeyStoreSpi.java

```diff
@@ -316,6 +319,8 @@ public abstract void engineStore(OutputStream stream, char[] password)
      *          algorithm could not be found
      * @throws    CertificateException if any of the certificates included in
      *          the keystore data could not be stored
+     * @throws UnsupportedOperationException if the implementation
```

Comment:
Please add an extra white space after the "throws" tag so that this line align with the lines above.

Reply:
On line 310 and 315, can you also fix the typos as part of this change: s/KeyStore.LoadStoreParmeter/KeyStore.LoadStoreParameter/

Reply:
Added the white space. Sure, I can include those typos fix in this PR. Also, included another typo fix for line 305.